src/main/java/ShortSet.java

```diff
@@ -4,11 +4,11 @@
   public static void main (String[] args) {
     Set<Short> s = new HashSet<>();
     for (short i = 0; i < 100; i++) {
-      s.add(i);
-      s.remove(i - 1);
+        short s = i - 1;
+        s.remove(s);
```

Comment:
Good job! The static analysis tool Error Prone reported an error [1] used to be here, but you fixed it. Check out http://errorprone.info for more information.


[1] ```05f655bbf3bfd8738007c2af426c001279cc8b56_src/main/java/ShortSet.java:8: error: [CollectionIncompatibleType] Argument 'i - 1' should not be passed to this method; its type int is not compatible with its collection's type argument Short
      s.remove(i - 1);
              ^
    (see http://errorprone.info/bugpattern/CollectionIncompatibleType)```